core/src/main/kotlin/com/tschuchort/compiletesting/AbstractKotlinCompilation.kt

```diff
@@ -36,8 +36,9 @@ abstract class AbstractKotlinCompilation<A : CommonCompilerArguments> internal c
     /** Working directory for the compilation */
     var workingDir: File by default {
         val path = Files.createTempDirectory("Kotlin-Compilation")
-        log("Created temporary working directory at ${path.toAbsolutePath()}")
-        return@default path.toFile()
+        val canonicalFile = path.toFile().canonicalFile
```

Comment:
The JS compiler calls .canonicalFile internally - at least on MacOS that may be a different to the absolute path (`/private/var/…` vs `/var/…`), which made the tests fail.